sykepenger-model/src/test/kotlin/no/nav/helse/person/InntektshistorikkVol2Test.kt

```diff
@@ -301,6 +301,7 @@ internal class InntektshistorikkVol2Test {
             1.desember(2016) til 1.august(2017) inntekter {
                 ORGNUMMER inntekt INNTEKT
             }
+            Thread.sleep(10) // Nødvendig for konsistent resultat på windows
```

Comment:
Jeg forstår ikke hvorfor dette fungerer, for er det ikke først i `forEach` under at det blir opprettet timestamp etterhvert?